indexing-service/src/main/java/io/druid/indexing/common/stats/RowIngestionMeters.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.indexing.common.stats;
+
+import com.codahale.metrics.Meter;
+import com.codahale.metrics.MetricRegistry;
+import com.google.common.collect.Maps;
+
+import java.util.Map;
+
+public class RowIngestionMeters
```

Comment:
without increasing the scope too much, can this be a bind-able `Interface` with a `DropwizardRowIngestionMeter` as the default bound thing? this would allow extensions to specify their own if desired.

Reply:
Essentially, this class would be renamed to `DropwizardRowIngestionMeter` and `RowIngestionMeters` would be an interface with the methods you need

Reply:
Sure, I'll look into making this extensible